buildenv

```diff
@@ -1,7 +1,10 @@
-export ZOPEN_GIT_URL=""
-export ZOPEN_TARBALL_URL="https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.gz"
-export ZOPEN_TARBALL_DEPS="make coreutils diffutils grep libtool zoslib"
-export ZOPEN_TYPE="TARBALL"
+# bump: libpcre2-version /PCRE2_VERSION="(.*)"/ gitrefs:https://github.com/PhilipHazel/pcre2.git|re:/pcre2-([\d.]+)/$1/|semver:*
+PCRE2_VERSION="10.42"
```

Comment:
This is looking at PCRE2, but it should be looking for PCRE1

Reply:
Oh ok, will fix..

Reply:
@IgorTodorovskiIBM Does this work?